utopia/components.utopia.js

```diff
@@ -356,7 +356,7 @@ const Components = {
       },
     },
     SpecialRow: {
-      component: SpecialRow,
+      component: ProductFeatureRow,
       properties: {
         left: {
```

Comment:
Shouldn't these be changes as well? (left/right)

Reply:
And the `code` field in `variants` as well

Reply:
Fixed!